TestON/drivers/common/cli/emulator/mininetclidriver.py

```diff
@@ -61,6 +62,22 @@ def connect( self, **connectargs ):
                 vars( self )[ key ] = connectargs[ key ]
 
             self.name = self.options[ 'name' ]
+
+            try:
```

Comment:
Sorry i forgot to document  it, ill push it again after the planning with complete documentation. but it  basically just grabs the os environment variables and set it as the ip address, its not completely structured yet but it works for now. The normal passing of the ip address is still working by the way. No change necessary in topo file for now.